data/json/requirements/toolsets.json

```diff
@@ -1,5 +1,23 @@
 [
   {
+    "id": "cooking_standard",
+    "type": "requirement",
+    "//": "Cooking of standard meals",
+    "qualities": [ { "id": "COOK", "level": 2 } ],
+    "tools": [ [
+      [ "fire", -1 ],
+      [ "hotplate", 10 ],
+      [ "mess_kit", 10 ],
+      [ "mil_mess_kit", 10 ],
+      [ "oil_cooker", 10 ],
+      [ "survivor_mess_kit", 10 ],
+      [ "gasoline_cooker", 10 ],
+      [ "charcoal_cooker", 10 ],
+      [ "popcan_stove", 10 ],
+      [ "toolset", 10 ]
+    ] ]
+  },
```

Comment:
> It is not granular enough, it could easily have 1 charge of heat items rather than 10

The rationale here is that for example we could scale `mil_mess_kit` to be more efficient than `mess_kit` and so on? The other obvious base units are `2`, `4` and `5`. I don't want to use `1` as this precludes any later tools with higher efficiency.

> The heat sources should be their own requirement, to actually fix the problem of having to repeat the tools rather than just mitigating it partially

The same applies for sewing, welding. We eventually need qualities consuming charges but this is better than what we currently have and also starts getting the recipes into a more consistent order.

> The quality shouldn't be bundled here, otherwise we'll need a separate copy of this requirement for easier to cook items

Fair enough. That group should just become `heat_source`?